views/js/runner/plugins/content/dialog/exitMessages.js

```diff
@@ -59,6 +59,12 @@ define([
                             testRunner.disableItem(context.itemIdentifier);
                         }
 
+                        // prevent all other messages which can overlap the dialog
+                        $('#feedback-box').addClass('hidden');
+                        setTimeout(function(){
+                            $('.feedback').remove();
+                            $('#feedback-box').removeClass('hidden');
+                        }, 3000);
```

Comment:
Could you show me this 'better way'? I don't know how to prevent different messages between plugins. I can create a new parameter in the config: `showTerminateFeedback: true`, but I'm not sure that it will make the system easier

Reply:
You could for instance move this stuff into the provider, listening to an event, lets says something like 'disablefeedbackalerts', and then simply trigger it from this component.
And instead of using a `setTimeout`, trigger `enablefeedbackalerts` once the dialog is closed.
That will have several advantages, at least from architecture point of view:
- share the feature with other components/plugins that could benefit of it
- centralize the feature at one place, simplifying any refactoring (I guess we could find a better approach to hide those feedbacks than directly relying on DOM)
- remove the soft coupling introduced here

The best solution would require to rework the feedbacks and the dialogs, to avoid this stacking issue. For instance, they could be placed on different areas: feedback always on the top, in the header area, dialogs always in the middle, in the content area. But this is a UX responsability.